### PR TITLE
Version 4.8.2

### PR DESCRIPTION
**Pull request by niknbr**
##### Checklist

- [x] `npm test` passes
- [ ] tests and/or benchmarks are included
- [x] commit message follows commit guidelines

### CODE REVIEW

**Comment by niknbr** [comment]
Closed for #55

**Comment by niknbr** [comment]
Two test failures

```
 FAIL  tests/unit/graph/graph-spec.ts
  ● Test suite failed to run

      ● redo stack & undo stack › fill undo stack

        expect(received).toBe(expected) // Object.is equality

        Expected: 1
        Received: 0

          1562 |     let undoStack = stackData.undoStack;
          1563 |     let redoStack = stackData.redoStack;
        > 1564 |     expect(undoStack.length).toBe(1);
               |                              ^
          1565 |     expect(undoStack[0].action).toEqual('render');
          1566 |     expect(undoStack[0].data.after.nodes.length).toEqual(2);
          1567 |     expect(redoStack.length).toBe(0);

      at Object.<anonymous> (tests/unit/graph/graph-spec.ts:1564:30)
```
☝️ Failing upstream. Caused by [this](https://github.com/antvis/G6/pull/4178/commits/69a861736ea35b6586cb5b89d51c85d6713005f6) change

```
     ● states › should change data and swap initial states on updated item if model has it
          expect(received).toEqual(expected) // deep equality
          - Expected  - 4
          + Received  + 1
          - Array [
          -   "appliedOnNodeNew",
          -   "multipleOnNodeNew:valueNew",
          - ]
          + Array []
            1746 |     });
            1747 |
          > 1748 |     expect(graph.findById('node1').getStates()).toEqual(['appliedOnNodeNew', 'multipleOnNodeNew:valueNew']);
                 |                                                 ^
            1749 |   });
            1750 |
            1751 |   it('should change data and apply initial styles on added item if model has it', () => {
      at Object.<anonymous> (tests/unit/graph/graph-spec.ts:1748:49)
```
☝️ Already failing on master. Not related to this PR.

**Review by Blakko — APPROVED** [review]

File: packages/pc/src/global.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ const textColor = 'rgb(0, 0, 0)';
 const colorSet = getColorsWithSubjectColor(subjectColor, backColor);
 
 export default {
-  version: '0.7.10-siren.0',
```
Comment (Blakko) [inline packages/pc/src/global.ts:10]: This should be updated

**Review by Blakko — CHANGES_REQUESTED** [review]

File: packages/pc/src/global.ts (hunk 1 above)
Comment (niknbr) [reply packages/pc/src/global.ts:10]: missed bump versions. Updated

**Review by Blakko — APPROVED** [review]

**Pull request merged**